README.md

```diff
@@ -4,13 +4,15 @@
 
 ## Latest Release: v2.13
 [Read the specification](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md)
+
 [Read the profile](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/profile.md)
 
-[Release Notes for past versions](release-notes.md)
+[Release Notes for past versions](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/release-notes.md)
 
 ## Working Changes
-[Read the working changes to the spec](spec.md)
-[Read the working changes to the profile](profile.md)
+[Read the working changes to the spec](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md)
+
+[Read the working changes to the profile](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md)
```

Comment:
I'm ok with the new lines, but I'd prefer to keep these relative so that local traversals work.

Reply:
That does mean though if somebody gets linked to v2.13 of the spec, then those links will show v2.13 of spec/profile, rather than master. Are we happy with that?

Reply:
What if instead of pointing to both docs individually (which could be kind of annoying since the # of docs may change after we cut a release) we just do something like:

[Read the latest working changes of the documentation.](https://github.com/openservicebrokerapi/servicebroker/)

Reply:
But then if I'm on the v2.13 branch, then I think that link would take me to a readme that looked exactly the same 😆

Reply:
yes, the latest doc would always point to itself if there's a link to the latest doc :-)